src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -1256,6 +1175,173 @@ static void glb_notify_on_state_change_locked(grpc_exec_ctx *exec_ctx,
       exec_ctx, &glb_policy->state_tracker, current, notify);
 }
 
+static void glb_update_locked(grpc_exec_ctx *exec_ctx, grpc_lb_policy *policy,
```

Comment:
It should not be necessary to move this function.  Please do the following:

- Put this back where it was originally.
- Put `glb_lb_policy_vtable` back where it was originally.
- Move `glb_create()` down to below the original location of `glb_lb_policy_vtable`.

In particular, note that it makes sense for `glb_create()` to be defined near the other methods in `glb_factory_vtable`, since that's where it's called from.  I would prefer to see all of the LB policy definitions before the LB policy factory definitions.

Reply:
Done